src/lib.rs

```diff
@@ -4,6 +4,10 @@
 //! [embedded-graphics](https://crates.io/crates/embedded-graphics) code, or produce examples and
 //! interactive demos to show of embedded graphics features.
 //!
+//! # [Examples](https://github.com/embedded-graphics/examples)
+//!
+//! Code using the simulator can be found in the [examples repository](https://github.com/embedded-graphics/examples).
+//!
```

Comment:
I believe this should have been removed.

Reply:
Thanks for catching that. Fixed now, along with reorganising the sections and a typo fix.